src/main/java/practice/StreamPractice.java

```diff
@@ -50,7 +67,12 @@ public Double getOddNumsAverage(List<Integer> numbers) {
      */
     public List<People> getWorkablePeople(int fromAge, int femaleToAge,
                                           int maleToAge, List<People> peopleList) {
-        return Collections.emptyList();
+        return peopleList.stream()
+                .filter(people -> people.getAge() >= fromAge && people.getAge() <= maleToAge
+                        && people.getSex().equals(People.Sex.MAN)
+                        || people.getAge() >= fromAge && people.getAge() <= femaleToAge
```

Comment:
```suggestion
                .filter(people -> people.getAge() >= fromAge && people.getAge() <= maleToAge
                        && people.getSex().equals(People.Sex.MAN)
                        ||  people.getAge() <= femaleToAge
```
we can check it only one time